sub/source/docs/rc-setup.rst

```diff
@@ -0,0 +1,36 @@
+.. _rc-setup:
+
+=============
+Radio Control
+=============
+
+Due to the RF attenuation of even a short distance of water above a submersible, radio control must be accomplished by floating tethered bouy to the vehicle which contains the RF components and antenna. This applies also to RF based telemetry, whether independent of, or integrated into, the radio control system.
```

Comment:
Possibly worth including [an explanatory link](https://en.wikipedia.org/wiki/Electromagnetic_absorption_by_water#Microwaves_and_radio_waves), for the curious? :-)

Reply:
TMI I think....they can explore if they want the "why"